docs/plugins/inputs/kafka.asciidoc

```diff
@@ -20,11 +21,24 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-This input will read events from a Kafka topic.
+This input will read events from a Kafka topic. It uses the 0.10 version of the
+consumer API provided by Kafka to read messages from the broker.
 
-This plugin uses Kafka Client 0.10.0.1. For broker compatibility, see the official https://cwiki.apache.org/confluence/display/KAFKA/Compatibility+Matrix[Kafka compatibility reference].
+Here's a compatibility matrix that shows the Kafka client versions that are compatible with each combination
```

Comment:
@jsvd Are we intentionally adding the matrix back in for 5.6, or is the doc build just pulling it in?  TBH, I've lost track, but I know this matrix has caused confusion.

Copying @jordansissel for clarification.

Reply:
my bad, this is because the plugin didn't make it into the release, so this doc page got overriden. I'll edit this